### PR TITLE
Add `cchardetect` CLI script

### PR DESCRIPTION
**Pull request by craigds**
boogie

### CODE REVIEW

File: bin/cchardetect [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+def read_chunks(f, chunk_size):
+    chunk = f.read(chunk_size)
+    while chunk:
+        yield chunk
+        chunk = f.read(chunk_size)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('files', nargs='+', help="Files to detect encoding of", type=argparse.FileType('r'))
```
Comment (PyYoshi) [inline bin/cchardetect]: it should be ``argparse.FileType('rb')`` because Python 3.x will not work.

**Comment by PyYoshi** [comment]
Hi @craigds 

Thanks a lot!!!!!!!!!!!!!!

[LGTM](https://lgtm.in/i/C7ifYM01q)

I left a comment on code. Please check it.

Thanks.

File: bin/cchardetect (hunk 1 above)
Comment (craigds) [inline bin/cchardetect]: good catch. fixed :)

**Comment by PyYoshi** [comment]
@craigds. Thank you!

**Pull request merged**